hw2/Index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw2</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<img src="img/logo.png">
+			<ul class="top-menu">
+				<li><a href="#">Главная</a></li>
+				<li><a href="#">Товары</a></li>
+				<li><a href="#">О_Нас</a></li>
+				<li><a href="#">Контакты</a></li>
+			</ul>	
+		</header>
+		<section class="container">
+			<div class="left-menu">
+				<ul>
+					<li><a href="#">Ноутбуки</a></li>
+					<li><a href="#">Планшеты</a></li>
+					<li><a href="#">Телефоны</a></li>
+					<li><a href="#">Телевизоры</a></li>
+					<li><a href="#">Бытовая техника</a></li>
+					<li><a href="#">Автотовары</a></li>
+				</ul>
+			</div>	
+			<div class="content">
+				<div class="image">
+					<img src="img/image.png">
+				</div>
+				<article class="article">
```

Comment:
article is good, but rather for the whole article with picture. Don't you think that picture belongs to this article?